Storage/SQL/BrowserTable.swift

```diff
@@ -822,7 +828,8 @@ open class BrowserTable: Table {
 
                 // Adds tables/indicies for metadata content
                 pageMetadataCreate,
-                indexPageMetadataCreate]) {
+                indexPageMetadataCacheKeyCreate,
```

Comment:
Double check to see that users who upgrade to 7.0 won't already be at version 19 of the database. We might need to add this addition to a version 20 migration.